package.json

```diff
@@ -36,7 +35,7 @@
     "eject": "react-app-rewired eject",
     "deploy": "yarn run build && gh-pages --add --dist build/",
     "deploy-dev":
-      "REACT_APP_POLLBOT_URL=$npm_package_pollbot_dev PUBLIC_URL=$npm_package_homepage_dev yarn run deploy --dest dev/",
+      "REACT_APP_POLLBOT_URL=$npm_package_pollbot PUBLIC_URL=$npm_package_homepage_dev yarn run deploy --dest dev/",
```

Comment:
This line should not be necessary. It's a repeat from the default in the code. 
Environment variables should (unless they're things like secrets) rarely be used if possible.

Reply:
indeed, updated ;)